File: serverSide/music/DAL/models/Album.cs
using System;
using System.Collections.Generic;

namespace DAL.models;

public partial class Album
{
    public int AlbumId { get; set; }

    public string AlbumName { get; set; } = null!;

    public int AlbumSingerId { get; set; }

    public DateTime? PublishingDate { get; set; }

    public DateTime? UploadDate { get; set; }

    public string? ImageUrl { get; set; }

    public virtual Singer AlbumSingerNavigation { get; set; } = null!;
}

File: clientSide/my-app/src/app/classes/Album.ts
export class Album {
  constructor(
    public albumId?: number,
    public albumName?: string,
    public albumSingerId?: number,
    public albumSingerFirstName?: string,
    public albumSingerLastName?: string,
    public publishingDate?: Date,
    public uploadDate?: Date,
    public imageUrl?: string
  ) {}
}
